packages/components/button/src/less/button.less

```diff
@@ -243,36 +243,25 @@ oui-button {
   min-width: 0;
   padding: @oui-button-link-padding;
   border: 0;
-  background: none;
-  color: @oui-button-link-font-color;
-  font-weight: @oui-button-link-font-weight;
-  text-decoration: @oui-button-link-text-decoration;
 
-  // stylelint-disable no-descending-specificity
-  .oui-icon {
-    color: @oui-button-link-icon-color;
-  }
-  // stylelint-enable no-descending-specificity
-
-  &:hover,
```

Comment:
Aren't you deleting a lot of features here ?

Reply:
I'm replacing it with a mixin, don't worry ;)